### PR TITLE
[Enterprise Search Module] Fixing the CI

### PR DESCRIPTION
**Pull request by afoucret**
Two small fixes to fix the CI of the ent-search-module PR.

### CODE REVIEW

File: x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/ClusterPrivilegeResolver.java [hunk 1]
```diff
@@ -153,7 +153,6 @@ public class ClusterPrivilegeResolver {
         RemoteClusterService.REMOTE_CLUSTER_HANDSHAKE_ACTION_NAME,
         RemoteClusterNodesAction.NAME
     );
-
```
Comment (ioanatia) [inline x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/ClusterPrivilegeResolver.java:156]: is this change needed?

Comment (afoucret) [reply x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/ClusterPrivilegeResolver.java:156]: CI was complaining about extra spaces at the end of this line. I restored the line as it is in main.

**Review by ioanatia — APPROVED** [review]
